### DISCUSSION TITLE
Connecting to supabase from github codespaces

### DISCUSSION THREAD

**Discussion by rodpatulski** [supabase/supabase#17442] [Questions]
I have a react project that tries to connect to supabase. It works fine, but I wanted to try it in github codespaces. 

I get the following error when I try to connect to supabase:
Access to manifest at 'https://github.dev/pf-signin?id=8xtp2drf&cluster=usw3&name=cautious-goggles-96w4rq47vprh9xgp&port=8100&pb=https%3A%2F%2Fcautious-goggles-96w4rq47vprh9xgp-8100.app.github.dev%2Fauth%2Fpostback%2Ftunnel%3Frd%3D%252Fmanifest.json%26tunnel%3D1' (redirected from 'https://cautious-goggles-96w4rq47vprh9xgp-8100.app.github.dev/manifest.json') from origin 'https://cautious-goggles-96w4rq47vprh9xgp-8100.app.github.dev/' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
This happens on the preflight request.

There is [this](https://supabase.com/docs/guides/functions/cors) doc page but it seems to deal with edge functions.

**Comment by ozanvos** [comment]
Hi @rodpatulski, were you able to get this working? Starting my Supabase dev server works but I'm getting CORS issues connecting to the auth api.

**Comment by rodpatulski** [comment]
@osener Sorry, i never got this working. I'm still holding out hope as it would be great to use codespaces. I wonder if this is a question better asked in the github codespaces discussion forum. Something tells me this is not just a supabase issue.
